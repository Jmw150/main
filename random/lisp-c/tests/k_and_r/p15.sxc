(#include <stdio.h>)

(#define LOWER 0)	// lower limit of table
(#define UPPER 200)	// upper limit
(#define STEP 20)	// step size

(int main ((int argc) (char (** argv)))
     (var int fahr)

     (for (= fahr LOWER) (<= fahr UPPER) (= fahr (+ fahr STEP))
	  (printf "%3d %6.1f\n" fahr (* (/ 5.0 9.0) (- fahr 32))))
     (return 0))
