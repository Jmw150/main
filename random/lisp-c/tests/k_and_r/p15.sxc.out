  0  -17.8
 20   -6.7
 40    4.4
 60   15.6
 80   26.7
100   37.8
120   48.9
140   60.0
160   71.1
180   82.2
200   93.3
